RELEASE.md

```diff
@@ -0,0 +1,5 @@
+Release type: patch
+
+Update the built-in GraphiQL to the current latest version 3.0.5.
+
+Update React to the current latest version 18.2.0 and use the production distribution instead of development to reduce bundle size.
```

Comment:
```suggestion
Release type: patch

This release updates the built-in GraphiQL to the current latest version (3.0.5), it also updates React to the current latest version (18.2.0) and uses the production distribution instead of development to reduce bundle size.
```